src/routes/user.routes.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+const router = express.Router();
+import { registerUser, loginUser } from "../controllers/user.controller.js";
+
+// 회원가입
+router.get("/join", registerUser);
```

Comment:
```suggestion
import userController from "../controllers/user.controller.js";

// 회원가입
router.get("/join", userController.registerUser);
```

이건 취향 차이일 수도 있지만, 요로콤 하면 import 문을 조금 더 간결하게 작성할 수 있을지도 모르겠네요 👀